nilearn/datasets/neurovault.py

```diff
@@ -969,15 +971,28 @@ def _get_batch(query, prefix_msg="", timeout=10.0, verbose=3):
     )
     prepped = session.prepare_request(req)
     logger.log(
-        f"{prefix_msg}getting new batch: {query}",
+        f"{prefix_msg}getting new batch:\n\t{query}",
         verbose=verbose,
         msg_level=_DEBUG,
-        stack_level=7,
+        stack_level=6,
     )
     try:
         resp = session.send(prepped, timeout=timeout)
         resp.raise_for_status()
         batch = resp.json()
+    except requests.exceptions.ReadTimeout:
```

Comment:
to test this, we would need to mock a time out error

I'd suggest that having a full build of the doc in main is higher priority, than the coverage, so I would rather fix this coverage issue in a follow up PR

Reply:
@jeromedockes you may know this from the top of your head, maybe... 🤞🏽

Reply:
all responses are already mocked, see https://github.com/nilearn/nilearn/blob/main/nilearn/datasets/tests/_testing.py

so you could do something like

```python
def test_error(request_mocker):
    import requests
    request_mocker.url_mapping["https://example.com"] = MyError()
    with pytest.raises(MyError):
        # replace with neurovault.fetch_something()
        requests.Session().send("https://example.com")

```